File: AndroidProyecto/app/src/main/java/com/example/androidappproyecto/modelos/Oferta.kt
package com.example.androidappproyecto.modelos

data class Oferta(
    var id: Int = 0,
    var cantidad: Double = 0.0,
    var descripcion: String? = null,

    var piso: Piso? = null,
    var inquilino: Inquilino? = null,

    var solicitudes: List<Solicitud>? = null

)

File: WinFormsAppProyecto/Modelos/Oferta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Modelos
{
    public class Oferta
    {
        public int id {  get; set; }
        public double cantidad {  get; set; }
        public string descripcion { get; set; }
        public Piso piso { get; set; }
        public Piso inqulino { get; set; }
    }
}
